### PR TITLE
php81Extensions.blackfire: 1.78.1 -> 1.84.0

### PR DESCRIPTION
**Pull request by shyim**
###### Description of changes

Update package

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [X] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by shyim** [comment]
@marsam Could you maybe merge this too? It is similar to https://github.com/NixOS/nixpkgs/pull/202932 ❤️

File: pkgs/development/tools/misc/blackfire/php-probe.nix [hunk 1]
```diff
@@ -18,52 +18,52 @@ let
...
   }.${phpMajor} or (throw "Unsupported PHP version.");
 
-  version = "1.78.1";
+  version = "1.84.0";
 
   hashes = {
     "x86_64-linux" = {
       system = "amd64";
-      sha256 = "Q9VuZewJ/KX2ZL77d3YLsE80B0y3RYg/hE2H14s9An4=";
+      sha256 = "4tAqe1ev2s4ZwzPptgXuVL4ZXF37ieGyonBxOFMUKTs=";
```
Comment (SuperSandro2000) [inline pkgs/development/tools/misc/blackfire/php-probe.nix:26]: Next time please don't strip prefixes from sri hashes.

Comment (shyim) [reply pkgs/development/tools/misc/blackfire/php-probe.nix:26]: With sri, you mean the prefix `sha256-` right?

Comment (SuperSandro2000) [reply pkgs/development/tools/misc/blackfire/php-probe.nix:26]: yes

**Pull request merged**